CI fails on CaseTrainBoostedTreesEstimatorAndExplain

Reference build: https://travis-ci.com/sql-machine-learning/sqlflow/jobs/274151235
```
=== RUN   TestEnd2EndMySQL/CaseTrainBoostedTreesEstimatorAndExplain
2020/01/10 07:42:53 runSQLProgram error: failed: exit status 1
==========Program==========
import tensorflow as tf
from sqlflow_submitter.tensorflow import explain
from tensorflow.estimator import DNNClassifier, DNNRegressor, LinearClassifier, LinearRegressor, BoostedTreesClassifier, BoostedTreesRegressor, DNNLinearCombinedClassifier, DNNLinearCombinedRegressor
try:
    import sqlflow_models
except:
    pass
feature_column_names = [
"sepal_length",
"sepal_width",
"petal_length",
"petal_width",
]
feature_metas = dict()
feature_metas["sepal_length"] = {
    "feature_name": "sepal_length",
    "dtype": "float32",
    "delimiter": "",
    "shape": [1],
    "is_sparse": "false" == "true"
}
feature_metas["sepal_width"] = {
    "feature_name": "sepal_width",
    "dtype": "float32",
    "delimiter": "",
    "shape": [1],
    "is_sparse": "false" == "true"
}
feature_metas["petal_length"] = {
    "feature_name": "petal_length",
    "dtype": "float32",
    "delimiter": "",
    "shape": [1],
    "is_sparse": "false" == "true"
}
feature_metas["petal_width"] = {
    "feature_name": "petal_width",
    "dtype": "float32",
    "delimiter": "",
    "shape": [1],
    "is_sparse": "false" == "true"
}
label_meta = {
    "feature_name": "class",
    "dtype": "int64",
    "delimiter": "",
    "shape": [],
    "is_sparse": "false" == "true"
}
model_params=dict()
model_params["center_bias"]=True
model_params["n_batches_per_layer"]=8
model_params["n_classes"]=2
model_params["n_trees"]=50
feature_columns = {"feature_columns": [tf.feature_column.numeric_column("sepal_length", shape=[1]),
tf.feature_column.numeric_column("sepal_width", shape=[1]),
tf.feature_column.numeric_column("petal_length", shape=[1]),
tf.feature_column.numeric_column("petal_width", shape=[1])]}
import json
summaryAttrs = json.loads('''{}''')
explain.explain(datasource="mysql://root:root@tcp(127.0.0.1:3306)/?maxAllowedPacket=0",
                estimator_cls=BoostedTreesClassifier,
                select="""SELECT * FROM iris.test WHERE class!=2
	""",
                feature_columns=feature_columns,
                feature_column_names=feature_column_names,
                feature_metas=feature_metas,
                label_meta=label_meta,
                model_params=model_params,
                save="model_save",
                result_table="iris.explain_result",
                hdfs_namenode_addr="",
                hive_location="",
                hdfs_user="",
                hdfs_pass="",
                **summaryAttrs)
==========Output==========
Traceback (most recent call last):
  File "<stdin>", line 97, in <module>
  File "/root/go/src/sqlflow.org/sqlflow/python/sqlflow_submitter/tensorflow/explain.py", line 76, in explain
    gain = estimator.experimental_feature_importances(normalize=True)
  File "/usr/local/lib/python3.6/dist-packages/tensorflow_estimator/python/estimator/canned/boosted_trees.py", line 1628, in experimental_feature_importances
    normalize)
  File "/usr/local/lib/python3.6/dist-packages/tensorflow_estimator/python/estimator/canned/boosted_trees.py", line 1461, in _compute_feature_importances
    assert normalizer > 0, 'Trees are all empty or contain only a root node.'
AssertionError: Trees are all empty or contain only a root node.
2020/01/10 07:42:53 stream read err: rpc error: code = Unknown desc = failed: exit status 1
==========Program==========
import tensorflow as tf
from sqlflow_submitter.tensorflow import explain
from tensorflow.estimator import DNNClassifier, DNNRegressor, LinearClassifier, LinearRegressor, BoostedTreesClassifier, BoostedTreesRegressor, DNNLinearCombinedClassifier, DNNLinearCombinedRegressor
try:
    import sqlflow_models
except:
    pass
feature_column_names = [
"sepal_length",
"sepal_width",
"petal_length",
"petal_width",
]
feature_metas = dict()
feature_metas["sepal_length"] = {
    "feature_name": "sepal_length",
    "dtype": "float32",
    "delimiter": "",
    "shape": [1],
    "is_sparse": "false" == "true"
}
feature_metas["sepal_width"] = {
    "feature_name": "sepal_width",
    "dtype": "float32",
    "delimiter": "",
    "shape": [1],
    "is_sparse": "false" == "true"
}
feature_metas["petal_length"] = {
    "feature_name": "petal_length",
    "dtype": "float32",
    "delimiter": "",
    "shape": [1],
    "is_sparse": "false" == "true"
}
feature_metas["petal_width"] = {
    "feature_name": "petal_width",
    "dtype": "float32",
    "delimiter": "",
    "shape": [1],
    "is_sparse": "false" == "true"
}
label_meta = {
    "feature_name": "class",
    "dtype": "int64",
    "delimiter": "",
    "shape": [],
    "is_sparse": "false" == "true"
}
model_params=dict()
model_params["center_bias"]=True
model_params["n_batches_per_layer"]=8
model_params["n_classes"]=2
model_params["n_trees"]=50
feature_columns = {"feature_columns": [tf.feature_column.numeric_column("sepal_length", shape=[1]),
tf.feature_column.numeric_column("sepal_width", shape=[1]),
tf.feature_column.numeric_column("petal_length", shape=[1]),
tf.feature_column.numeric_column("petal_width", shape=[1])]}
import json
summaryAttrs = json.loads('''{}''')
explain.explain(datasource="mysql://root:root@tcp(127.0.0.1:3306)/?maxAllowedPacket=0",
                estimator_cls=BoostedTreesClassifier,
                select="""SELECT * FROM iris.test WHERE class!=2
	""",
                feature_columns=feature_columns,
                feature_column_names=feature_column_names,
                feature_metas=feature_metas,
                label_meta=label_meta,
                model_params=model_params,
                save="model_save",
                result_table="iris.explain_result",
                hdfs_namenode_addr="",
                hive_location="",
                hdfs_user="",
                hdfs_pass="",
                **summaryAttrs)
==========Output==========
Traceback (most recent call last):
  File "<stdin>", line 97, in <module>
  File "/root/go/src/sqlflow.org/sqlflow/python/sqlflow_submitter/tensorflow/explain.py", line 76, in explain
    gain = estimator.experimental_feature_importances(normalize=True)
  File "/usr/local/lib/python3.6/dist-packages/tensorflow_estimator/python/estimator/canned/boosted_trees.py", line 1628, in experimental_feature_importances
    normalize)
  File "/usr/local/lib/python3.6/dist-packages/tensorflow_estimator/python/estimator/canned/boosted_trees.py", line 1461, in _compute_feature_importances
    assert normalizer > 0, 'Trees are all empty or contain only a root node.'
AssertionError: Trees are all empty or contain only a root node.
FAIL	sqlflow.org/sqlflow/cmd/sqlflowserver	34.989s
```

--- a/cmd/sqlflowserver/main_test.go
+++ b/cmd/sqlflowserver/main_test.go
@@ -677,11 +677,10 @@ func CaseTrainBoostedTreesEstimatorAndExplain(t *testing.T) {
 	SELECT * FROM iris.train WHERE class!=2
 	TO TRAIN BoostedTreesClassifier
 	WITH
-		model.n_batches_per_layer=10,
-		model.n_trees=50,
+		model.n_batches_per_layer=1,
 		model.center_bias=True,
-		train.batch_size=10,
-		train.epoch=20,
+		train.batch_size=100,
+		train.epoch=10,
 		validation.select="SELECT * FROM iris.test where class!=2"
 	LABEL class
 	INTO %s;

--- a/cmd/sqlflowserver/main_test.go
+++ b/cmd/sqlflowserver/main_test.go
@@ -677,11 +677,10 @@ func CaseTrainBoostedTreesEstimatorAndExplain(t *testing.T) {
 	SELECT * FROM iris.train WHERE class!=2
 	TO TRAIN BoostedTreesClassifier
 	WITH
-		model.n_batches_per_layer=8,
+		model.n_batches_per_layer=10,
 		model.n_trees=50,
-		model.n_classes=2,
 		model.center_bias=True,
-		train.batch_size=8,
+		train.batch_size=10,
 		train.epoch=20,
 		validation.select="SELECT * FROM iris.test where class!=2"
 	LABEL class

